packages/build/bin/mocha-current-file.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
```

Comment:
If this file is only used for `loopback-next` repo, please move it to `loopback-next/bin`.

Reply:
Done!